docs/mddocs/Quickstart/benchmark_quickstart.md

```diff
@@ -80,22 +80,14 @@ Some parameters in the yaml file that you can configure:
 
 Please refer to [here](../Overview/install_gpu.md#runtime-configuration) to configure oneAPI environment variables. Choose corresponding commands base on your device.
 
-- For **Intel iGPU**:
```

Comment:
Directly change this to

- For **Intel iGPU** and **Intel Arc™ A-Series Graphics*